android/src/ti/map/TiMarker.java

```diff
@@ -74,4 +74,13 @@ public String getSnippet()
 		}
 		return null;
 	}
+
+	@Override
+	public Float getZIndex()
```

Comment:
Is this documented?

Reply:
it's nothing in the module we use. It's an override that we have to implement to be able to use the the map-utils and return the z-value if available.
This is their example https://github.com/googlemaps/android-maps-utils/blob/46bca380e930ff9723d3a5266e2e3c81fb9a20c4/demo/src/main/java/com/google/maps/android/utils/demo/model/MyItem.java#L55

Reply:
Mhh, okay. And it does not run without that? Thats a pretty massive breaking change in their API then

Reply:
> TiMarker is not abstract and does not override abstract method getZIndex() in ClusterItem

when I remove it

Reply:
Okay, then it's indeed a breaking change on their end. If the z-index still behaves correctly, this is fine

Reply:
and yes, it's marked as a breaking change:

> BREAKING CHANGES
>   Add z-index to cluster markers (https://github.com/googlemaps/android-maps-utils/issues/1136) ([2a74367](https://github.com/googlemaps/android-maps-utils/commit/2a743675804585eb3749ef11392edc7c5de4c4bc))
> Apologies for breaking semantic versioning; only realized after release that the commit was not marked for breaking change.